### PR TITLE
Dict update

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32022,6 +32022,7 @@ ipmrovement->improvement
 ipmrovements->improvements
 ipmroves->improves
 ipmroving->improving
+iproving->improving
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32025]: Time permitting, I would suggest adding all verbal forms:
```suggestion
iprove->improve, I prove,
iproved->improved, I proved,
iproves->improves
iproving->improving
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -40527,6 +40528,7 @@ pincher->pinscher
 pinnaple->pineapple
 pinockle->pinochle
 pinoneered->pioneered
+pint->point
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:40531]: [`pint`](https://www.oed.com/search/dictionary/?q=pint) is a word and cannot be added to the typos dictionary.

Comment (macpijan) [reply codespell_lib/data/dictionary.txt:40531]: true, thanks

Comment (macpijan) [reply codespell_lib/data/dictionary.txt]: This commit has been dropeed

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (macpijan) [reply codespell_lib/data/dictionary.txt:32025]: Extended, thanks for suggestion

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**